src/ResumeCache.cpp

```diff
@@ -62,6 +62,11 @@ void ResumeCache::trackSentFrame(
     // TODO(tmont): this could be expensive, find a better way to get length
     auto frameDataLength = serializedFrame.computeChainDataLength();
 
+    if (streamIdPtr) {
```

Comment:
this should probably switch on `frameType` and only do this for `REQUEST_*` frames.